src/lib/ai/providers.ts

```
@@ -1,30 +1,39 @@
+const hackclub = createOpenAICompatible({
+  name: 'hackclub',
+  apiKey: env.HACKCLUB_API_KEY,
+  baseURL: 'https://ai.hackclub.com',
+});
+
+const openrouter = createOpenRouter({
+  apiKey: env.OPENROUTER_API_KEY!,
+});
+
+const google = createGoogleGenerativeAI({
+  apiKey: env.GOOGLE_GENERATIVE_AI_API_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)